app/assets/javascripts/modules/case_worker/admin/management_information.js

```diff
@@ -0,0 +1,63 @@
+moj.Modules.ManagementInformation = {
+  init: function () {
+    this.bindEvents();
+  },
+
+  bindEvents: function () {
+    this.provisionalAssessmentsDateFieldChange();
+    this.provisionalAssessmentsDownloadClicked();
+  },
+
+  buildDataArray: function() {
+    var data = {};
+    data.api_key='64c9177b-e5a0-4ac5-9f2b-33166597c436';
```

Comment:
is this key ok here?

Reply:
Good catch... that should have been current_user.api_key... I wonder where that went! 😨